features/taxonomy-management/org.wso2.carbon.governance.taxonomy.management.server.feature/pom.xml

```diff
@@ -0,0 +1,166 @@
+        <dependency>
+            <groupId>commons-scxml.wso2</groupId>
+            <artifactId>commons-scxml</artifactId>
+        </dependency>
+    </dependencies>
+    <build>
+        <plugins>
+
+            <plugin>
+                <artifactId>maven-resources-plugin</artifactId>
+                <version>2.6</version>
+                <executions>
+                    <execution>
+                        <id>prefilter-resources</id>
+                        <phase>generate-resources</phase>
+                        <goals>
+                            <goal>copy-resources</goal>
+                        </goals>
+                        <configuration>
+                            <outputDirectory>src/main/resources</outputDirectory>
+                            <resources>
+                                <resource>
+                                    <directory>resources</directory>
+                                    <includes>
+                                        <include>**/*</include>
+                                        <include>p2.inf</include>
+                                        <include>build.properties</include>
+                                    </includes>
+                                </resource>
+                            </resources>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.wso2.maven</groupId>
+                <artifactId>carbon-p2-plugin</artifactId>
+                <version>${carbon.p2.plugin.version}</version>
+                <executions>
+                    <execution>
+                        <id>4-p2-feature-generation</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>p2-feature-gen</goal>
+                        </goals>
+                        <configuration>
+                            <id>org.wso2.carbon.governance.taxonomy.management.server</id>
+                            <propertiesFile>../../etc/feature.properties</propertiesFile>
+                            <adviceFile>
+                                <properties>
+                                    <propertyDef>org.wso2.carbon.p2.category.type:server</propertyDef>
+                                    <propertyDef>org.eclipse.equinox.p2.type.group:false</propertyDef>
+                                </properties>
+                            </adviceFile>
+                            <bundles>
+                                <bundleDef>org.wso2.carbon.governance:org.wso2.carbon.governance.custom.lifecycles.checklist
+                                </bundleDef>
+                                <bundleDef>org.wso2.carbon.governance:org.wso2.carbon.governance.taxonomy</bundleDef>
+                                <bundleDef>org.wso2.carbon.governance:org.wso2.carbon.governance.registry.eventing</bundleDef>
```

Comment:
do we need registry.eventing bundle and governance.custom.lifecycles.checklist bundle to be included in this feature?